modules/vaos/app/services/vaos/jwt_wrapper.rb

```diff
@@ -24,11 +24,11 @@ def token
     def payload
       {
         authenticated: true,
-        sub: icn,
+        sub: @user.icn,
```

Comment:
Same here, since there's an `attr_reader` in this class, these can all refer to `user` without the symbol